Lib/sqlite3/test/dbapi.py

```diff
@@ -160,6 +160,15 @@ def CheckInTransactionRO(self):
         with self.assertRaises(AttributeError):
             self.cx.in_transaction = True
 
+    def CheckOpenWithPathLikeObject(self):
+        """ Checks that we can succesfully connect to a database using an object that
+            is PathLike, i.e. has __fspath__(). """
+        self.addCleanup(unlink, TESTFN)
+        import pathlib
+        path = pathlib.Path(TESTFN)
```

Comment:
I'm not sure that it's ok to depend on pathlib when testing sqlite.

I propose to replace it with:

```
class Path(os.PathLike):
    def __fspath__(self):
        return TESTFN
```